lib/components/atoms/Blockquote/Blockquote.style.js

```diff
@@ -0,0 +1,35 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  position: relative;
+  ${props => `
+      padding: ${props.theme.spacing.unit1} ${props.theme.spacing.unit9};
+    `};
+  .content {
+    ${props => `
+        ${props.theme.typography.FONT_H2};
+        padding: ${props.theme.spacing.unit1}
+      `};
+  }
+  .author {
+    text-align: right;
+    ${props => `
+      ${props.theme.typography.FONT_H3};
+      padding: ${props.theme.spacing.unit1}
+    `};
+  }
+  &:before {
+    display: block;
+    height: 0;
+    content: '”';
+    position: absolute;
+    top: 0;
+    ${props => `
+      ${props.theme.typography.FONT_QUOTE};
```

Comment:
Rename this to FONT_QUOTE --> FONT_QUOTE_SIGN

Reply:
fixed